src/components/API-calls.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const BaseURL = "https://my-backend-server-josh.herokuapp.com/api";
```

Comment:
This isn't quite how they intended the baseURL thing to be used, but if it works, it works. 

Here's a link if you want to look at the other way, using axios.create
https://stackoverflow.com/questions/64798213/how-to-configure-axios-base-url